lib/suspenders/app_builder.rb

```diff
@@ -154,6 +158,11 @@ def replace_gemfile
       template 'Gemfile.erb', 'Gemfile'
     end
 
+    def replace_gemfile_mongoid
+      remove_file 'Gemfile'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.